CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
  * Upgraded in #217:
    * Kafka client version to 2.3.1: https://kafka.apache.org/23/documentation.html#upgrade_230_notable
    * Confluent Platform components version to 5.3.1: https://docs.confluent.io/5.3.1/release-notes/index.html#cp-5-3-1-release-notes
+ * Added functions to simply querying the test machine journal (#215)
```

Comment:
Think "simply" might be a typo here. Did you mean "simplify" or maybe "support"?

Reply:
Oh yeah thanks I meant 'simplify' ... although happy to change it to 'support' if you think that reads better

Reply:
Simplify is fine with me :-)